posthog/session_recordings/ai_data/ai_regex_schema.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum
+from pydantic import BaseModel
+
+
+class ResultEnum(str, Enum):
```

Comment:
if it's useful...

We can add these to `schema.ts` (or maybe that's a folder now)
and that gets autoconverted to a python schema, so we can use the same types in the front and backend